projects/netgrif-components-core/src/lib/data-fields/case-ref-field/case-ref-default/case-ref-default.component.ts

```diff
@@ -59,7 +65,7 @@ export abstract class AbstractCaseRefDefaultComponent extends AbstractBaseDataFi
                 },
                 {
                     provide: NAE_BASE_FILTER,
-                    useValue: { filter: SimpleFilter.fromCaseQuery({stringId: filterValue}) } as BaseFilter
+                    useValue: { filter: SimpleFilter.fromCaseQuery((filterProperty && query ? query : {stringId: filterValue})) } as BaseFilter
```

Comment:
So if the `filterProperty` is true and no `query` is provided, query by `stringId` is applied. Is this behavior desired?